test/tests/CMakeLists.txt

```diff
@@ -174,7 +174,7 @@ if (NOT DISABLE_NETWORK)
     add_executable(test_crypt "${CMAKE_CURRENT_LIST_DIR}/CryptTests.cpp"
                               "${CMAKE_CURRENT_LIST_DIR}/TestData.cpp")
     SET_CHECK_CXX_FLAGS(test_crypt)
-    target_link_libraries(test_crypt ${GTEST_LIBRARIES} libopenrct2)
+    target_link_libraries(test_crypt ${GTEST_LIBRARIES} test-common libopenrct2)
```

Comment:
This doesn't look right. Why is it needed here, when `libopenrct2` is already present?